test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -168,22 +169,27 @@ try
     Install-OpenCover -TargetDirectory $openCoverTargetDirectory -force
     Write-LogPassThru -Message "OpenCover installed."
 
-    Write-LogPassThru -Message "TestDirectory : $testPath"
+    Write-LogPassThru -Message "TestPath : $testPath"
     Write-LogPassThru -Message "openCoverPath : $openCoverTargetDirectory\OpenCover"
     Write-LogPassThru -Message "psbinpath : $psBinPath"
     Write-LogPassThru -Message "elevatedLog : $elevatedLogs"
     Write-LogPassThru -Message "unelevatedLog : $unelevatedLogs"
     Write-LogPassThru -Message "TestToolsPath : $testToolsPath"
 
     $openCoverParams = @{outputlog = $outputLog;
-        TestDirectory = $testPath;
+        TestPath = $testPath;
```

Comment:
since these hashtable entries are on separate lines, the `;` is not really needed. I don't think it's a big deal, but something to watch in future